src/main/java/core/basesyntax/strategy/FruitTransactionStrategy.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.strategy.impl.BalanceTransaction;
+import core.basesyntax.strategy.impl.PurchaseTransaction;
+import core.basesyntax.strategy.impl.ReturnTransaction;
+import core.basesyntax.strategy.impl.SupplyTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitTransactionStrategy {

```

Comment:
To follow Single Responsibility divide this class into two. One will deal with parsing of data and another one with strategy